### DISCUSSION TITLE
Considering hybrid assembly with metaspades or just using metaflye.  How does metaspades handle overlapping long reads that do not overlap with small reads?

### DISCUSSION THREAD

**Discussion by jolespin** [ablab/spades#1227] [Q&A] [answered]
> hybrid assembly for metagenomes remains an experimental pipeline and optimal performance is not guaranteed.

If I have pretty deeply sequenced long reads and average depth illumina reads, would you recommend using metaspades with the --nanopore option or metaflye? I have deeply sequenced nanopore reads (7-9 GB gzipped) and Illumina reads (~2-3 GB gzipped).

For long reads that do not overlap with many short reads, are they assembled separately into contitgs/scaffolds or are they removed in spades/metaspades?

**Comment by asl** [comment, marked as answer]
SPAdes is a short-read assembler and uses long reads to supplement short read assembly, it does not assemble the. You can read the details in hybridSPAdes article: https://academic.oup.com/bioinformatics/article/32/7/1009/1743807

**Reply by jolespin** [reply]
Ok awesome, thank you.  This is what I was expecting but just wanted to make sure.
